Sorts/CombSort.java

```diff
@@ -0,0 +1,64 @@
+// Java program for implementation of Comb Sort
+
+class CombSort
+{
+    // To find gap between elements
+    int getNextGap(int gap)
```

Comment:
You can this methods declare as ```static```. If you do this you can the methods right away use in the main-method.